libraries/standard/mqtt/cbmc/proofs/MQTT_GetIncomingPacketTypeAndLength/Makefile

```diff
@@ -0,0 +1,14 @@
+HARNESS_ENTRY=harness
+HARNESS_FILE=MQTT_GetIncomingPacketTypeAndLength_harness
+
+DEFINES +=
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY +=
+UNWINDSET += getRemainingLength.0:5
+
+PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c
+PROOF_SOURCES += $(PROOFDIR)/../../stubs/NetworkInterfaceReceiveStub.c
+PROJECT_SOURCES += $(SRCDIR)/libraries/standard/mqtt/src/mqtt_lightweight.c
+
+include ../Makefile.common
```

Comment:
All files should end with a newline.

And copyright

```
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0
```

Reply:
True, but this is also auto-generated, so the script needs to make it end in a new line?